js/src/forum/components/Search.js

```diff
@@ -71,8 +71,11 @@ export default class Search extends Component {
     // Hide the search view if no sources were loaded
     if (!this.sources.length) return <div></div>;
 
+    const searchCtaText = extractText(app.translator.trans('core.forum.header.search_placeholder'));
```

Comment:
What is cta? Can we just say label or placeholder?

Reply:
CTA means call to action.

I would have done that but it's a label and a placeholder, rather than one or the other.

Reply:
In other parts of the codebase, we use `label` to describe placeholders. Let's use label: even if not as descriptive, it's clearer.

Reply:
Done! :)